src/main/java/edu/harvard/iq/dataverse/engine/command/impl/CreateNewDataFilesCommand.java

```diff
@@ -639,6 +641,24 @@ public CreateDataFileResult execute(CommandContext ctxt) throws CommandException
                 }
                 logger.fine("Supplied type: " + suppliedContentType + ", finalType: " + finalType);
             }
+            if (newFileSize == null) {
```

Comment:
@landreev - is this still needed? I think I added it when you needed the size here for quotas.

Reply:
No, it should not be needed. The code currently in develop should be doing the right thing (assume that it's ok if the size is null, and not bother trying to look it up).